docs/monitoring/monitoring.md

```diff
@@ -17,225 +18,195 @@ The following topics are covered in this section:
 
 * [Dashboard](#dashboard)
 
-* [Member Details View](#member-details)
+* [Jobs](#jobs)
 
-* [SQL Page](#sql)
+* [Stages](#stages)
 
-* [Jobs Page](#jobs)
+* [Spark Cache](#spark_cache)
 
-* [Stages Page](#stages)
+* [Environment](#environment)
 
-<a id="dashboard"></a>
-## The Dashboard
-
-The Dashboard offers the following capabilities and benefits:
+* [Executors](#executors)
 
-* [Cluster Statistics](#cluster)
+* [SQL](#sql)
 
-* [Member Statistics](#member)
+!!! Note
+	When using Smart Connector with upstream Spark, the **Dashboard** section is not displayed. Only the **SQL**, **Jobs** and **Stages** related information is displayed.
 
-* [Table Statistics](#table)
+On the top-right side of the SnappyData Pulse page, you can view the version details of SnappyData Snapshot. When you click this, the name and version of the product, the build details, the source revision details and the version number of the underlying spark are displayed.
+![](../Images/snapshot.png)
 
-* [External Table Statistics](#external-table)
+You can also view the total number of CPU cores allocated to your cluster on the top-right side of the page.
+![](../Images/CPU_cores.png)
+Click this and the number of cores allocated to each member is displayed.
+![](../Images/CPU_cores1.png)
```

Comment:
Remove this image which tells user member wise CPU core numbers.
Also remove line "Click this and the number of cores allocated to each member is displayed." 

Now we only display Total CPU Cores count.